commands/runner.go

```diff
@@ -188,3 +198,26 @@ func splitAliasCmd(cmd string) ([]string, error) {
 
 	return words, nil
 }
+
+func getListOfBuiltInCommands() map[string]bool {
+	gitHelpOutput, _ := cmd.New("git help -a").CombinedOutput()
+	builtInCommands := parseGitHelpOutput(gitHelpOutput)
+	for hubCommand, _ := range CmdRunner.All() {
+		builtInCommands[hubCommand] = true
+	}
+	return builtInCommands
+}
+
+func parseGitHelpOutput(gitHelpOutput string) map[string]bool {
```

Comment:
If you move this function to the `git/` package, then you can use `gitOutput("help", "-a")` and get output broken into lines. Then you can iterate through those lines and have a simpler regex that just verifies that a line starts with exactly 2 spaces, and `Split()` the whole line without having to `FindAllString`.